int64 bodymotionid
---

bool pose_data_capture